integrations/amazon_bedrock/src/haystack_integrations/common/amazon_bedrock/utils.py

```diff
@@ -20,6 +21,7 @@ def get_aws_session(
     aws_session_token: Optional[str] = None,
     aws_region_name: Optional[str] = None,
     aws_profile_name: Optional[str] = None,
+    async_client: bool = False,
```

Comment:
I would call this param only `async` since this method returns a `Session` (and then you get the client later using `Session.client(...)`. WDYT?

Reply:
I can change to some other name, `async` is a reserved keyword

Reply:
yeah you're right... what about `use_async_session` then?